server/src/schemas/stats/boards.stats.schema.ts

```diff
@@ -0,0 +1,60 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document, Types } from 'mongoose';
+import { StatsUserTimestamp } from 'src/shared/interfaces/StatsUserTimestamp';
+import { Schema as MongooseSchema } from 'mongoose';
+
+@Schema()
```

Comment:
if you add this before your schema, createdAt and updatedAt props will be automatically added 

```
@Schema({
  timestamps: true,
  versionKey: false,
  validateBeforeSave: true,
})
```

Reply:
There is no use for this fields.